packages/@lwc/integration-karma/scripts/karma-configs/test/sauce.js

```diff
@@ -88,12 +26,6 @@ module.exports = (config) => {
     const sauceConfig = getSauceConfig(config, {
         suiteName: 'integration-karma',
         tags: TAGS,
-        customData: {
-            lwc: {
-                COMPAT,
-                DISABLE_SYNTHETIC: !SYNTHETIC_SHADOW_ENABLED,
-            },
-        },
```

Comment:
I dunno what this was doing. [According to SauceLabs](https://docs.saucelabs.com/basics/test-config-annotation/test-annotation/), it is "User-defined custom data that will accept any valid JSON object, limited to 64KB in size." I assume it just goes in the Circle CI UI somewhere.